app/serializers/post_serializer.rb

```diff
@@ -12,6 +12,7 @@ class PostSerializer < ActiveModel::Serializer
   )
 
   belongs_to :category
+  belongs_to :serial
```

Comment:
Yes, we want to know posts belong to serial or not.
Because we should navigate users to a post's serial page.

----

その記事が連載に紐付いていたらその連載への導線を作りたいので、そこで欲しくなるんですよー。